docs_template/source/guides_explanations/environments.md

```diff
@@ -4,101 +4,67 @@ of your favourite data analysis library. The same holds for debugging errors tha
+or when one manually runs `pixi install`. The environment is downloaded into the hidden
+folder `.pixi` in your project root.
 
 ### Setting up a new environment
 
-If you want to create a clean environment we recommended specifying it through an
-environment.yml file. Below we show the contents of an example environment.yml file. A
-detailed explanation is given in the
-[Conda documentation](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#create-env-file-manually).
-
-```yaml
-name: <env_name>
-
-channels:
-  - conda-forge
-  - nodefaults
-
-dependencies:
-  - python=3.12
-  - numpy
-  - pandas
-  - pip
-```
+A pixi environment is always created on a per-project basis. That means, it only makes
+sense to create a new environment, if you create a new project.
 
-If the environment.yml file exists you can create the environment using
-
-```console
-$ conda create -f path/to/environment.yml
-```
+If you want to do this, you can simply follow the instructions in the
+[template setup guide](../getting_started/template_setup.md).
 
 ### Updating packages
 
-Make sure you activated the environment by `conda activate <env_name>`. Then run
+Make sure that you are in the project root and run
 
 ```console
-$ conda update [package]
+$ pixi update
 ```
 
-to update a specific `[package]`, or run
+to update all packages, or run
 
 ```console
-$ conda update --all
+$ pixi update [package]
 ```
 
-to update all packages.
+to update a specific `[package]`.
 
 ### Installing additional packages
 
 To list installed packages, activate the environment and type
 
 ```console
-$ conda list
+$ pixi list
 ```
 
-If you want to add a package to your environment, add it to the environment.yml file.
-Once you have edited the environment.yml file, run
+If you want to add a package to your environment, you can add the package to the
+`[tool.pixi.dependencies]` section in the pyproject.toml file. Alternatively, you can
+run
 
 ```console
-$ mamba env update -f environment.yml
+$ pixi add [package]
 ```
 
-**Choosing between conda and pip**
-
-Generally it is recommended to use *conda* whenever possible. It is a necessity for many
-scientific packages. These often are not pure-Python code and pip is built mainly for
-that. For pure-Python packages, sometimes nobody bothered to set up a conda package and
-we use *pip*.
+You will notice that the pixi section in the pyproject.toml file is then also updated
+with the added package.
 
-If you add a package under `dependencies:` in the environment.yml file, conda will try
-to install its own package. If you add a package under `pip:`, conda will try to install
-the package via pip.
+**Choosing between conda-forge and PyPI**
 
-### Information about your conda environments
-
-For listing your installed conda environments, type
-
-```console
-$ conda info --envs
-```
+Generally it is recommended to use *conda-forge* whenever possible. It is a necessity
```

Comment:
I think we never explained this, did we?

*Ahhh, we need to swap this paragraph with the next one.*